components/pages/NFT/NFT.tsx

```diff
@@ -43,10 +44,12 @@ const NFTPage: React.FC<NFTPageProps> = ({
   user,
   setUser,
   type,
+  isUserFromDappQR,
 }) => {
   const [likeLoading, setLikeLoading] = useState(false);
   const [modalShareOpen, setModalShareOpen] = useState(false);
   const [byTheSameArtistNFTs, setByTheSameArtistNFTs] = useState<NftType[]>([])
+  const isVR = (NFT.categories.findIndex(x => x.code === "vr") !== -1 || NFT.serieId === "1390370908") && NFT.creator === NFT.owner
```

Comment:
Is this `NFT.serieId === "1390370908"` check required ? What does this serieId represent ?